DemoServer/client/src/store/reducers/tracking.ts

```diff
@@ -22,6 +22,11 @@ export function trackingReducer(state: TrackingState = initialState,
             return modifyState(state, s => {
                 s.showConstentMonit = true;
             });
+            
+        case "TRACKING_HIDE_MONIT":
+            return modifyState(state, s => {
+                s.showConstentMonit = false;
+        });
```

Comment:
invalid indent

Reply:
https://media.giphy.com/media/10LkbTpwgPzWLu/giphy.gif

Reply:
Thanks for another one!